storage/remote/queue_manager_test.go

```diff
@@ -842,7 +842,7 @@ func createTimeseries(numSamples, numSeries int, extraLabels ...labels.Label) ([
 	return samples, series
 }
 
-func createProtoTimeseriesWithOld(numSamples, baseTs int64, _ ...labels.Label) []prompb.TimeSeries {
+func createProtoTimeseriesWithOld(numSamples, baseTs int64, extraLabels ...labels.Label) []prompb.TimeSeries {
```

Comment:
Did you intend to use this parameter?

Reply:
It isn't used in any call anymore, so I believe we can just remove it.